pkg/executor/aggfuncs/aggfuncs.go

```diff
@@ -139,8 +139,21 @@ const (
 // to be any type.
 type PartialResult unsafe.Pointer
 
+// AggPartialResultMapper contains aggregate function results
+type AggPartialResultMapper map[string][]PartialResult
+
+type serializer interface {
+	// SerializePartialResult will serialize meta data of aggregate function into bytes and put them into chunk.
+	SerializePartialResult(ctx sessionctx.Context, partialResult PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper)
+
+	// DeserializePartialResult deserializes from bytes to PartialResult.
+	DeserializePartialResult(ctx sessionctx.Context, src *chunk.Chunk) ([]PartialResult, int64)
```

Comment:
Is the `ctx` input argument really needed?
```suggestion
	SerializePartialResult(partialResult PartialResult, chk *chunk.Chunk, spillHelper *SpillSerializeHelper)

	// DeserializePartialResult deserializes from bytes to PartialResult.
	DeserializePartialResult(src *chunk.Chunk) ([]PartialResult, int64)
```

Reply:
> Is the `ctx` input argument really needed?

I don't know as they have been here before this pr.

Reply:
It can be removed if it's not used

Reply:
> It can be removed if it's not used

done